<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadFilesWindow</class>
 <widget class="QMainWindow" name="UploadFilesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>517</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Horus File Upload</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QProgressBar" name="pbCurrentFile">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblCurrentFile">
         <property name="text">
          <string>Current File: </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frameQueue">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QHBoxLayout" name="layoutQueueLabel">
         <item>
          <widget class="QLabel" name="lblQueue">
           <property name="text">
            <string>Queued For Upload</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblZipSwitch">
           <property name="text">
            <string>Zip Multiple Files</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="lvFiles"/>
    </item>
    <item>
     <widget class="QLabel" name="lblCompleted">
      <property name="text">
       <string>Completed (click entry to copy URL)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="lvCompleted"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="sb"/>
 </widget>
 <resources/>
 <connections/>
</ui>
